posthog/api/test/test_insights.py

```diff
@@ -0,0 +1,220 @@
+import json
+from datetime import datetime, timedelta
+
+from dateutil.relativedelta import relativedelta
+from django.test.utils import override_settings
+from django.utils import timezone
+from freezegun import freeze_time
+
+from posthog.ee import is_ee_enabled
+from posthog.models.dashboard_item import DashboardItem
+from posthog.models.event import Event
+from posthog.models.filter import Filter
+from posthog.models.person import Person
+from posthog.test.base import TransactionBaseTest
+from posthog.utils import relative_date_parse
+
+# TODO: two tests below fail in EE
```

Comment:
Is this comment new or old? Is there an issue to link to here?

Reply:
Old, copypasta.

Reply:
No context on this issue from me.